src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -120,8 +120,11 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
   private def startRunnableCalls(): State = {
     tryStartingRunnableCalls() match {
       case Success(entries) =>
-        if (entries.nonEmpty) tryStartingRunnableCalls()
-        goto(WorkflowRunning)
+        if (entries.nonEmpty) {
+          startRunnableCalls()
+        } else {
+          goto(WorkflowRunning)
```

Comment:
Wait I don't undestand this function at all...

If it fails, we try starting runnable calls again but this time ignore the return value and don't transition...?